mxcube3/routes/qutils.py

```diff
@@ -301,18 +301,39 @@ def _handle_char(sample_id, node):
     queueID = node._node_id
     enabled, state = get_node_state(queueID)
 
+    originID, task = _handle_diffraction_plan(node)
     res = {"label": "Characterisation",
            "type": "Characterisation",
            "parameters": parameters,
            "checked": node.is_enabled(),
            "sampleID": sample_id,
            "taskIndex": node_index(node)['idx'],
            "queueID": node._node_id,
-           "state": state
+           "state": state,
+           "diffractionPlan": task,
+           "diffractionPlanID": originID
```

Comment:
I made a general comment about it this earlier. Its perhaps better if the counteraction contains  a list of all the collections in the diffraction plan ie diffractionPlan: [task1, task2 ... taskn]

We can like this display them in the chracterisation entry and the user can select
which one they like to use if not all.